packages/warehouses/package.json

```diff
@@ -15,7 +15,7 @@
         "lodash": "^4.17.21",
         "pg": "^8.13.1",
         "pg-cursor": "^2.10.0",
-        "snowflake-sdk": "~2.1.3",
+        "snowflake-sdk": "~2.3.1",
```

Comment:
there was a bug about `connectAsync` on 2.1.3 (version from 2020)  

>  When we tried to use external browser authentication with
  connectAsync(), we got the error: "External browser and Okta are not compatible with connection
  process"

I don't see any breaking changes https://docs.snowflake.com/en/release-notes/clients-drivers/nodejs-2025